tests/builtins/test_ascii.py

```diff
@@ -2,7 +2,33 @@
 
 
 class AsciiTests(TranspileTestCase):
-    pass
+    def test_ascii_string_without_unicode(self):
+        self.assertCodeExecution("""
+            print(ascii("hello world"))
+            print(ascii(chr(130)))
+            print(ascii(chr(270)))
```

Comment:
I think it would be nice to at least test all the 'boundary' values – so based on the code above, we have to make sure that it changes around 127/128 and 255/256 and so on, so the test could check those values. I'm curious if it prints some of the low (`< \x20`) non-printable ASCII characters as well.

Another approach might be to just generate a long string that includes every single character from 0 to n, and then let the test suite compare the resulting `ascii(string)` output. That should ensure that every character is encoded to the appropriate format.